zuul.d/jobs.yaml

```diff
@@ -0,0 +1,37 @@
+---
+- job:
+    name: base-crc-openstack
+    parent: base-crc
+    timeout: 10800
+    nodeset:
+      nodes:
+        name: controller
+        label: centos-9-stream-crc-xl
+    required-projects: &required_projects
+      - openstack-k8s-operators/install_yamls
+      - opendev.org/zuul/zuul-jobs
+      - openstack-k8s-operators/openstack-operator
+      - name: openstack-k8s-operators/openstack-ansibleee-operator
+        override-checkout: main
+      - name: openstack-k8s-operators/dataplane-operator
+        override-checkout: main
+      - name: openstack-k8s-operators/repo-setup
+        override-checkout: main
```

Comment:
Zuul befault looks for master branch. But each of these projects have default branch named 'main'. that's why we need to specify that otherwise zuul will fail with RETRY_LIMIT during job workspace preparation.

Reply:
I copy pasted it from previous job, will remove it. Thank for making great point here. these projects are already not needed.